app/controllers/site_controller.rb

```diff
@@ -67,6 +87,14 @@ def load_global
     @sdgs = YAML.load(File.open("#{Rails.root}/lib/data/content/sdgs.yml", 'r'))
   end
 
+  def generateImageUrls targets
+    targets['list'].each do |target|
```

Comment:
You should be able to use `map` instead of `each` so you won't need to return `targets` at the end. `map` returns the modified array.

Reply:
Ah, I see the point now. That's fine then!